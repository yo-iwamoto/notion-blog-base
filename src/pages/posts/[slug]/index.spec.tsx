import { render } from '@/__tests__/util';
import Page from './index.page';

describe('[page] /tags', () => {
  it('renders /tags unchanged', async () => {
    const { container } = render(
      <Page
        title='title'
        tags={[
          {
            id: 'id',
            color: 'blue',
            name: 'React',
          },
        ]}
        blocks={
          [
            // TODO
          ]
        }
        created_at='2022/02/20'
      />
    );

    expect(container).toMatchSnapshot();
  });
});
