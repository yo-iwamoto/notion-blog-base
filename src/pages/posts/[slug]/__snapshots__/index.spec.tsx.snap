// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[page] /tags renders /tags unchanged 1`] = `
<div>
  <div
    class="font-sans text-gray-900"
  >
    <div
      class="mx-4 mt-12"
    >
      <div
        class="mx-auto w-full max-w-2xl"
      >
        <main>
          <h1
            class="text-3xl font-bold"
          >
            title
          </h1>
          <div
            class="h-4"
          />
          <div
            class="flex gap-2"
          >
            <span
              class="cursor-pointer before:content-[\\"#\\"] hover:text-gray-500"
            >
              React
            </span>
          </div>
          <div
            class="h-2"
          />
          <p>
            2022/02/20
          </p>
          <div
            class="h-10"
          />
        </main>
      </div>
    </div>
  </div>
</div>
`;
